PowerEditor/src/WinControls/TaskList/TaskList.cpp

```diff
@@ -205,18 +211,19 @@ LRESULT TaskList::runProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lParam)
 			else
 			{
 				int32_t selected = (_currentIndex + 1) > (_nbItem - 1) ? 0 : (_currentIndex + 1);
-				ListView_SetItemState(_hSelf, _currentIndex, 0, LVIS_SELECTED|LVIS_FOCUSED);
+				ListView_SetItemState(_hSelf, _currentIndex, 0, LVIS_SELECTED|LVIS_FOCUSED);				

```

Comment:
thanks for the tips. Testing on Win7 shows that if the mouse is outside the task list it jumps 2 items up or down instead of just 1.  So let me fix that up and Ill submit another pr, ill be sure to eliminate the formatting issues as well.